src/app/components/PublicLayoutWrapper/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+
+export default function PublicLayout({ children }) {
```

Comment:
`export default function PublicLayoutWrapper({ children }) {`

remember that this will be the suggested name by the IDLE